framework/aster-frame/src/arch/x86/boot/boot.S

```diff
@@ -17,6 +17,8 @@ ENTRYTYPE_LINUX_64      = 4
 MULTIBOOT_ENTRY_MAGIC   = 0x2BADB002
 MULTIBOOT2_ENTRY_MAGIC  = 0x36D76289
 
+KERNEL_VMA              = 0xffffffff80000000
```

Comment:
VMA = Virtual Memory Area? This is a Linux term. Asterinas does not have this concept. Could you give this address another name that is most fitting in the context of Asterinas?

Reply:
It was just copied from `x86_64.ld` in fact.

What about `KERNEL_LOADED_OFFSET`? It's named after
https://github.com/asterinas/asterinas/blob/main/framework/aster-frame/src/vm/kspace.rs#L49-L51

Reply:
I see. I guess VMA stands for virtual memory address. So it is acceptable, although this abbreviation is seldomly used in Asterinas. So it is not a particularly good abbreviation. Let's keep the name for now.